app/charts/column/columns-grouped-state.tsx

```diff
@@ -267,23 +266,19 @@ const useGroupedColumnsState = (
 
     // y
     const minValue = Math.min(
-      mkNumber(
-        min(preparedData, (d) =>
-          getYErrorRange ? getYErrorRange(d)[0] : getY(d) || 0
-        )
-      ),
+      min(preparedData, (d) =>
+        getYErrorRange ? getYErrorRange(d)[0] : getY(d)
+      ) ?? 0,
       0
     );
     const maxValue = Math.max(
       max(preparedData, (d) =>
-        getYErrorRange ? getYErrorRange(d)[1] : getY(d) || 0
-      ) as number,
+        getYErrorRange ? getYErrorRange(d)[1] : getY(d)
+      ) ?? 0,
       0
     );
 
-    const yScale = scaleLinear()
-      .domain([mkNumber(minValue), mkNumber(maxValue)])
-      .nice();
+    const yScale = scaleLinear().domain([minValue, maxValue]).nice();
```

Comment:
It is because were are sure that values in preparedData are numbers and not strings ?

Reply:
It just wasn't necessary to use the function, as above it we already made sure `minValue` & `maxValue` are numbers (`... || 0`). I changed the logic slightly to use `?? 0` :)